spec/lib/day_hour_string_spec.rb

```diff
@@ -0,0 +1,52 @@
+# Copyright 2011-2022, The Trustees of Indiana University and Northwestern
+#   University.  Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+#   under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+#   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+#   specific language governing permissions and limitations under the License.
+# ---  END LICENSE_HEADER BLOCK  ---
+
+require 'rails_helper'
+
+describe 'HumanReadableDuration' do
+  describe 'to_human_readable_s' do
```

Comment:
```suggestion
describe 'DayHourString' do
  describe 'to_day_hour_s' do
```
I apparently overlooked changing the `describe` lines when I changed the name.